src/routers/InitializeDevSessionRouter.ts

```diff
@@ -22,7 +22,8 @@ class InitializeDevSessionRouter extends ApplicationRouter<SerializedUserSession
     }
 
     try {
-      const user = await UserRepo.createDummyUser('googleId');
+      const { firstName, googleID, lastName, netID } = req.query
```

Comment:
Let's make this pass in the information through JSON in the request body instead of a query, since we have so many keys.

Reply:
also add trailing semicolon!